project_closing/README.rst

```diff
@@ -0,0 +1,54 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===============
+Project closing
+===============
+
+This module extends the functionality of projects to allow automatic
+account analytic archiving when related project is archived, and if
+a project is dearchived, the related analytic account is dearchived as well.
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Go to *Project > Project > Projects*
+* Edit one
+* Click on *Archive*
+* The associated analytic account will also be archived.
+
```

Comment:
Add this lines:
.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
   :alt: Try me on Runbot
   :target: https://runbot.odoo-community.org/runbot/project/11.0